packages/utils/src/peripherals/relayer.ts

```diff
@@ -0,0 +1,114 @@
+import { BigNumber, constants } from "ethers";
+
+import { getHardcodedGasLimits } from "../constants";
+import { getChainIdFromDomain, getDecimalsForAsset } from "../helpers";
+import { Logger } from "../logging";
+
+import { ChainData } from "./chainData";
+import { getConversionRate, getGelatoEstimatedFee } from "./gelato";
+
+const relayerBufferPercentage = 20; // 20% bump on total estimated relayer fee
+
+export const calculateRelayerFee = async (
+  params: {
+    originDomain: string;
+    destinationDomain: string;
+    originNativeToken?: string;
+    destinationNativeToken?: string;
+    callDataGasAmount?: number;
+    isHighPriority?: boolean;
```

Comment:
What does the high priority arg mean exactly?

Also it would be nice to make this a separate type (could go in this same file here I think?)

Reply:
high priority is used to calculate fee using gelato api.